contrib/backporting/Dockerfile

```diff
@@ -1,17 +1,19 @@
 FROM ubuntu:20.04
 
 LABEL maintainer="maintainer@cilium.io"
+ARG GOLANG_VERSION=1.19.3
```

Comment:
Will this get updated by `make update-go-version` as well? If not, it that `make` target should be extended to update it.

Reply:
Good point @tklauser, I wasn't aware of that target in the Makefile.
I updated the PR!